gaphor/diagram/copypaste.py

```diff
@@ -212,7 +212,8 @@ def element_lookup(ref: str):
             return new_elements[ref]
 
         looked_up = lookup(ref)
-        if looked_up:
+        # TODO: Presentation element should be created anyhow
```

Comment:
Sorry to always  bring up these, but is there anything we need to do with this TODO before merging?

Reply:
You're right :smiley: . I hadn't noticed this one.